packages/static-build/test/fixtures/eleventy-2/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eleventy-base-blog",
+  "version": "5.0.0",
+  "description": "A starter repository for a blog web site using the Eleventy static site generator.",
+  "scripts": {
+    "build": "eleventy",
+    "watch": "eleventy --watch",
+    "debug": "DEBUG=* eleventy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/11ty/eleventy-base-blog.git"
+  },
+  "author": {
+    "name": "Zach Leatherman",
+    "email": "zachleatherman@gmail.com",
+    "url": "https://zachleat.com/"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/11ty/eleventy-base-blog/issues"
+  },
+  "homepage": "https://github.com/11ty/eleventy-base-blog#readme",
+  "devDependencies": {
+    "@11ty/eleventy": "^2.0.0",
+    "@11ty/eleventy-plugin-rss": "^1.2.0",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^5.0.0",
+    "luxon": "^1.12.0",
+    "markdown-it": "^8.4.2",
+    "markdown-it-anchor": "^5.0.2"
+  }
+}
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add lockfile

In this or a follow-up PR: Can we add a lockfile for this fixture?‏

Reply:
I was mystified by why it didn't have one, but its own generator maybe wants to ignore them? https://github.com/vercel/vercel/pull/10656/commits/dc7d64a117dc87fb6d0c9eeae4c7c0b8ce8a2422#diff-1ea7fca44fbf9989e209c50074eab7e51e440fb5dc53beed3e5a5a26a17b9632

Added now